systemtests/tests/python-dir-plugin-test/python-modules/bareos-dir-test.py

```diff
@@ -0,0 +1,74 @@
+# BAREOS - Backup Archiving REcovery Open Sourced
+#
+# Copyright (C) 2019-2019 Bareos GmbH & Co. KG
+#
+# This program is Free Software; you can redistribute it and/or
+# modify it under the terms of version three of the GNU Affero General Public
+# License as published by the Free Software Foundation, which is
+# listed in the file LICENSE.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.
+#
+# Author: Tobias Plum
+#
+from bareosdir import *
+from bareos_dir_consts import *
+
+def load_bareos_plugin(context, plugindef):
+  events = []
+  events.append(bDirEventType['bDirEventJobStart'])
+  events.append(bDirEventType['bDirEventJobEnd'])
+  events.append(bDirEventType['bDirEventJobInit'])
+  events.append(bDirEventType['bDirEventJobRun'])
+  RegisterEvents(context, events)
+  return bRCs['bRC_OK']
+
+def parse_plugin_definition(context, plugindef):
+  plugin_options = plugindef.split(":")
+  for current_option in plugin_options:
+    key,sep,val = current_option.partition("=")
+    if val == '':
+      continue
+    elif key == 'output':
+      global outputfile
+      outputfile = val
+      continue
+    elif key == 'instance':
+      continue
+    elif key == 'module_path':
+      continue
+    elif key == 'module_name':
+      continue
+    else:
+      return bRCs['bRC_Error']
+    toFile(outputfile)
+
+  return bRCs['bRC_OK']
+
+def handle_plugin_event(context, event):
+  if event == bDirEventType['bDirEventJobStart']:
+    toFile("bDirEventJobStart\n")
+
+  elif event == bDirEventType['bDirEventJobEnd']:
+    toFile("bDirEventJobEnd\n")
+
+  elif event == bDirEventType['bDirEventJobInit']:
+    toFile("bDirEventJobInit\n")
+
+  elif event == bDirEventType['bDirEventJobRun']:
+    toFile("bDirEventJobRun\n")
+
+  return bRCs['bRC_OK']
+
+def toFile(text):
+  doc = open(outputfile,"a")
+  doc.write(text)
+  doc.close()
```

Comment:
while most things in Bareos are indented with 2 spaces, python has an official style guide (PEP-8) that has been applied to the rest of our Python code.
Consider reformatting using `black` or at least make sure your indentation uses 4 spaces.

Reply:
I adopted the indentation from the templates in the core. Should I reformat them with black too?

https://github.com/bareos/bareos/blob/master/core/src/plugins/dird/bareos-dir.py.template

Reply:
You could do so, but please use a separate commit or even a separate PR.

Reply:
A separate commit would be good for that, yes.

Reply:
Should I reformat all python scripts in the repository or just the templates for the plugins?

Reply:
PR: #322